### PR TITLE
💄 MT 828: Decrease max length on text

### PR DESCRIPTION
**Pull request by itemodahl**

### CODE REVIEW

**Review by lha047 — CHANGES_REQUESTED** [review]
I guess you have something in the web project as well? 
The ´.children-full-width´ should be full width as described by the class name. Individual children elements can take up less space by individual targeting.

Have Malin look at the sizing before PR

File: src/scss/components/_top-tasks.scss [hunk 1]
```diff
@@ -3,6 +3,7 @@
 .top-tasks-container {
   h2 + .top-tasks-text {
     margin-top: var(--spacer-xx-small);
+    max-width: 47.5rem;
```
Comment (lha047) [inline src/scss/components/_top-tasks.scss:6]: This should not be neccesary. The `col-x-span-x` should do this work.

File: src/scss/components/_article.scss [hunk 2]
```diff
@@ -30,6 +30,7 @@
   @media only screen and (min-width: $breakpoint-large) {
     > * {
       grid-column: 3 / span 8;
+      max-width: 47.5rem;
```
Comment (lha047) [inline src/scss/components/_article.scss:33]: This should not be neccesary. The `grid-column: 3 / span 8;` should do this work.

File: stories/components/content/TopTasks.stories.svelte [hunk 3]
```diff
@@ -55,7 +55,7 @@
       {@html title}
     </h2>
     {#if intro}
-      <div class="top-tasks-text col-1-span-7">{@html intro}</div>
+      <div class="top-tasks-text col-1-span-6">{@html intro}</div>
```
Comment (lha047) [inline stories/components/content/TopTasks.stories.svelte:58]: Remember to do this update in the web-project as well :)

File: stories/components/content/LegalGuidanceCollection.stories.svelte [hunk 4]
```diff
@@ -71,7 +71,7 @@
 
       {#each legalItems as legal}
         <article
-          class="legal-collection legal-collection__border-top col-3-span-8"
+          class="legal-collection legal-collection__border-top col-3-span-6"
```
Comment (lha047) [inline stories/components/content/LegalGuidanceCollection.stories.svelte:74]: Remember to do this update in the web-project as well :)

**Review by lha047 — CHANGES_REQUESTED** [review]
Nice :D 
Have a look at the comments

**Comment by itemodahl** [comment]
This was added in https://github.com/Mattilsynet/designsystem/pull/402

**Pull request closed without merging**